keadm/app/cmd/util/centosinstaller.go

```diff
@@ -76,6 +76,22 @@ func (c *CentOS) InstallDocker() error {
 	return nil
 }
 
+func (c *CentOS) InstallContainerd() error {
+	fmt.Println("InstallContainerd called")
+	// yum install -y yum-utils
```

Comment:
please remove commented code, also there is an open pr for centos installer. let's handle this later after the centos installer code is merged.